README.md

```diff
@@ -326,6 +326,56 @@ Translations of the guide are available in the following languages:
   end
   ```
 
+<a name="multi-condition-case-when"></a>
+* put multiple when conditions on separate lines
+particularly where the conditions form long, complicated lines
+<sup>[[link](#multi-condition-case-when)]</sup>
+
+  ```Ruby
+  # good
+
+  case token
+  when :star_op
```

Comment:
What about keeping examples tight and informative? (ruby-style-guide already have more than 3.5k lines of text)
The first 2 similar `when`s can be merged:
```ruby
case token
when :star_op
  stack.pop * stack.pop
when :minus_op, :minus_minus_op
  also_calculate_that
  stack.pop - stack.pop
when MyModule::SomeDomain::BETA_USERS,
     MyModule::SomeDomain::INTERNAL_RELEASE
  stack.pop + stack.pop
when :int_literal,
     :some_complicate_explicit_name,
     :graduate_borrowers_with_arms,
     :str_interpolated 
  token.value
end